internal/generator/fluentd/conf_test.go

```diff
@@ -537,7 +537,7 @@ var _ = Describe("Testing Complete Config Generation", func() {
   </filter>
   
   # Include Infrastructure logs
-  <match kubernetes.var.log.pods.openshift-*_** kubernetes.var.log.pods.default_** kubernetes.var.log.pods.kube-*_** journal.** system.var.log**>
+  <match kubernetes.var.log.pods.openshift kubernetes.var.log.pods.openshift-*_** kubernetes.var.log.pods.default_** kubernetes.var.log.pods.kube-*_** journal.** system.var.log**>
```

Comment:
will this match something in openshift namespace?  it may still need `_**` in order to match

Reply:
I don't think so, according to this [wildcards-expansions-and-other-tips](https://docs.fluentd.org/configuration/config-file#wildcards-expansions-and-other-tips), will match `openshift` workspace

Reply:
But yes, looks like you are right, need to add `_**`.